Mathlib/Data/DFinsupp/Basic.lean

```diff
@@ -166,7 +166,7 @@ theorem mapRange_comp (f : ∀ i, β₁ i → β₂ i) (f₂ : ∀ i, β i → 
     (hf₂ : ∀ i, f₂ i 0 = 0) (h : ∀ i, (f i ∘ f₂ i) 0 = 0) (g : Π₀ i : ι, β i) :
     mapRange (fun i => f i ∘ f₂ i) h g = mapRange f hf (mapRange f₂ hf₂ g) := by
   ext
-  simp only [mapRange_apply]; rfl
+  simp only []; rfl
```

Comment:
I suspect all `simp only []` could be replaced by `dsimp only` (or at least `simp only`)

Reply:
In this case it seems that just `rfl` works.

I'm starting to get ambitious about this. Why not write a whole bunch of golfers like the ones I already have: One to remove lemmas from simps, one to convert `simp only []` to `dsimp only`, one to convert `simp only` to `simp_rw`, and many more?